elastic/kibana
# [Mappings editor] Support for dynamic templates

This PR adds support for dynamic templates within the mappings editor.

### Screenshots

<img width="1194" alt="Screen Shot 2020-01-11 at 11 56 03 AM" src="https://user-images.githubusercontent.com/5226211/72207703-87a26a00-3469-11ea-9289-e8f22395534a.png">

<img width="1196" alt="Screen Shot 2020-01-11 at 11 56 24 AM" src="https://user-images.githubusercontent.com/5226211/72207704-883b0080-3469-11ea-9c4d-784125c14922.png">

### How to test
- Verify you can add/edit dynamic templates data through the mappings editor
- Verify validation occurs for invalid JSON
- Verify you are able to add dynamic templates data through the  "Load from JSON" button.
